### PR TITLE
Updated info pull request

### PR DESCRIPTION
**Pull request by killshott3r**
Please accept this pull req, its for the updated info. It will be updated further, but for now host this one.

### CODE REVIEW

**Comment by srevinsaju** [comment]
@killshott3r any particular reason why the functions folders was renamed?

File: index.html [hunk 1]
```diff
@@ -55,22 +55,29 @@
...
+          color: #003165;
+          padding: 10px 15px;
+        }
+  </style>
   <header id="header" class="fixed-top">
 
     <div class="container">
-
-      <div class="logo float-left">
+      <div class="logo float-left" >
```
Comment (srevinsaju) [inline index.html:69]: This change was redundant.

File: index.html [hunk 2]
```diff
@@ -284,109 +292,134 @@ <h3>Councils</h3>
...
-              <h4 class="title"><a href="">US Senate</a></h4>
-              <p class="description">United States Senate Council</p>
-              <a href="https://mega.nz/#!3X5QTATC!807NckbcI1tn--Sy3IU05BieLx9V_odSQB3iQ-JcAPw" class="description btn-get-started scrollto">Get Background Guide</a>
-            </div>
-          </div>
+          <style>
+                img{margin-left:0.8pc;}
+                .col-md-6{margin-left: 5pc;}
+                @media only screen and (max-width: 600px) {.col-md-6{margin-left: 0pc;}}
+              </style>
```
Comment (srevinsaju) [inline index.html:299]: Also, please move this to `main.css`. 
Preferably use css classes to put them in `main.css`

```css
.myawesomeimage {
  margin-left: 0.8pc;
}
```
and then 
```html
<img class="myawesomeimage" ....>
```

File: index.html [hunk 3]
```diff
@@ -284,109 +292,134 @@ <h3>Councils</h3>
...
-          <div class="col-md-8 col-lg-5 offset-lg-1 wow bounceInUp" data-wow-duration="1.4s">
+          <div class="col-md-8 col-lg-5  wow bounceInUp" data-wow-duration="1.4s">
             <div class="box intro-info">
-              <div class="icon"><div class="sochum"></div></div>
-              <h4 class="title"><a href="">SOCHUM</a></h4>
-              <p class="description">Social, Humanitarian, and Cultural Committee</p>
-              <a href="https://mega.nz/#!Ge5UVQaa!HZ7SO62b1lMKaaFIiSxGmvS6SH163PRDSB23-PZoYN4" class="description btn-get-started scrollto">Get Background Guide</a>
+              <div class="icon"><div class="sochum"><img src="https://upload.wikimedia.org/wikipedia/commons/5/55/Emblem_of_India.svg" alt="" height=72px></div></div>
+              <h4 class="title"><a href="">Lok Sabha</a></h4>
+              <p class="description">Parliment of India</p>
```
Comment (srevinsaju) [inline index.html:315]: I think there is typo here.

File: index.html [hunk 4]
```diff
@@ -474,199 +507,784 @@ <h4><a href="#">App 1</a></h4>
...
+  <div class="modal-dialog modal-dialog-centered" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLongTitle">Prince Prakash</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        Prince Prakash is a Grade 11, Science student of The New Indian School. His first MUN experience was in his 7th grade and he was hesitant in the initial parts of the conference. But as soon as he participated in his first Challenge against a senior upon Chair’s discretion, he got all worked up. He later bagged the ‘Best Speaker’ award in the same conference. Ever since, he has found MUNs as the right platform to bring out the best among students in terms of thinking, debating and discussing. He also feels that every student should take part in MUNs, as it develops them to think on a global level and find solutions to real world problems. He also believes that a MUN experience will help delegates to unleash the potential they never knew they had.
```
Comment (srevinsaju) [inline index.html:709]: Should `challenge` be in lowercase here?

**Review by srevinsaju — CHANGES_REQUESTED** [review]

**Comment by killshott3r** [comment]
> @killshott3r any particular reason why the functions folders was renamed?

I don't remember changing that, it's probably my bad memory.

**Comment by srevinsaju** [comment]
Looks like a visual inconsitency:
Previous: 
image

Now: 
image

**Comment by srevinsaju** [comment]
Video embed height looks incorrect 

image

**Comment by killshott3r** [comment]
> Video embed height looks incorrect 
> 
> image
> 

Yeah that was something I didn't fix, will do later

**Comment by killshott3r** [comment]
> Looks like a visual inconsitency:
> Previous: 
> image
> 
> Now: 
> image
> 

Now this one I don't know how to fix

File: index.html [hunk 5]
```diff
@@ -1570,4 +2196,3 @@ <h4>Our Newsletter</h4>
   <script src="js/main.js"></script>
 
 </body>
-</html>
```
Comment (srevinsaju) [inline index.html:1573]: You removed the closing tag of the html document.

**Review by srevinsaju — CHANGES_REQUESTED** [review]

**Comment by srevinsaju** [comment]
Superseded by #20

**Pull request closed without merging**